config/twill.php

```diff
@@ -218,6 +218,12 @@
     'locale' => 'en',
     'fallback_locale' => 'en',
 
+    'locales' => [
+        'en' => 'English',
+        'es' => 'Spanish',
+    ],
+    'locale_picker_active' => true,
```

Comment:
The `locale_picker_active` config value doesn't exist anywhere in our code or any vendor packages. This can be removed.